### PR TITLE
natalie 0.5.0

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by apjanke** [comment]
Looks like the distribution layout has changed.

Failure:

```
==> Using the sandbox
/usr/bin/sandbox-exec -f /tmp/homebrew20161002-35836-114i4vf.sb nice /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/build.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/natalie.rb --build-bottle --verbose
==> Downloading https://github.com/krzyzanowskim/Natalie/archive/0.5.0.tar.gz
Already downloaded: /Users/brew/Library/Caches/Homebrew/natalie-0.5.0.tar.gz
==> Verifying natalie-0.5.0.tar.gz checksum
tar xzf /Users/brew/Library/Caches/Homebrew/natalie-0.5.0.tar.gz
Error: No such file or directory - (natalie.swift, natalie-script.swift)
```

Looks like that is caused by:
```
  def install
    mv "natalie.swift", "natalie-script.swift"
```

Unclear what to do about this. It looks like this version is more of a binary release.

**Comment by dunn** [comment]
Should be able to follow these build instructions instead https://github.com/krzyzanowskim/Natalie#swift-package-manager

**Comment by ilovezfs** [comment]
PR refreshed.

File: Formula/natalie.rb [hunk 1]
```diff
@@ -14,18 +14,13 @@ class Natalie < Formula
   depends_on :xcode => "7.0"
```
Comment (MikeMcQuaid) [inline Formula/natalie.rb]: Random comment: does using swift still always require Xcode?

**Comment by MikeMcQuaid** [comment]
Yosemite 💥. I guess Swift is too old: https://bot.brew.sh/job/Homebrew%20Core/9599/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_natalie/

**Comment by ilovezfs** [comment]
Without Xcode:
```
==> swift build -c release -Xswiftc -static-stdlib
error: unable to invoke subcommand: /Library/Developer/CommandLineTools/usr/bin/swift-build (No such file or directory)
```

**Pull request closed without merging**